bindata/network/ovn-kubernetes/007-flowschema.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
+kind: FlowSchema
+metadata:
+  name: openshift-ovn-kubernetes
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
```

Comment:
This annotation only makes sense for things in `manifests/` (so should be removed both here and from the sdn one)

Reply:
ACK, will change.